### PR TITLE
2470 cleanup

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/views/FileDocument/SelectDocumentWizardOverlay/ViewDocumentCategory.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const ViewDocumentCategory = connect(
                 id="view-all-document-subheader"
                 tabIndex="-1"
               >
-                Select Document Type
+                Select Document type
```
Comment (kkoskelin) [inline web-client/src/views/FileDocument/SelectDocumentWizardOverlay/ViewDocumentCategory.jsx:60]: Wondering if the rules dictate this should be `Select document type`?

File: web-client/src/views/RequestAccess/PartiesRepresenting.jsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const PartiesRepresenting = connect(
           >
             <fieldset className="usa-fieldset margin-bottom-0">
               <legend className="with-hint" id="who-legend">
-                Who Are You Representing?
+                Who are you Representing?
```
Comment (kkoskelin) [inline web-client/src/views/RequestAccess/PartiesRepresenting.jsx:37]: `...representing?` ?  If I'm wrong about this, please help me out and tell me where to review the capitalization rules so I can do better. :)

**Review by sutt0n — APPROVED** [review]

File: web-client/src/views/RequestAccess/RequestAccessDocumentReadOnly.jsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ export const RequestAccessDocumentReadOnly = connect(
           {requestAccessHelper.showFilingNotIncludes && (
             <div className="usa-form-group">
               <label className="usa-label" htmlFor="filing-not-includes">
-                Filing Does Not Include
+                Filing does not include:
```
Comment (kkoskelin) [inline web-client/src/views/RequestAccess/RequestAccessDocumentReadOnly.jsx:96]: was it requested to add a colon to this string?
Also in this same file, line 114 seems to use a different capitalization scheme; unsure if it needs to be changed?

**Review by wbyoko — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]
Just a couple of observations and questions; approving because I really don't know if I'm right or wrong. ;)

**Pull request merged**